wiki/Article_Styling_Criteria/en.md

```diff
@@ -82,150 +100,232 @@ tags:
 
 ### Folder names
 
-Folder names must never contain URL reserved characters. Generally, all folder names should only use these characters:
+Folder names must never contain URL reserved characters. Generally, all folder names must use these characters:
 
 - A - Z (uppercase letters)
 - a - z (lowercase letters)
 - 0 - 9 (numbers)
 - `_` (underscore)
 - `!` (exclamation mark)
 
+*Notice: Spaces must be changed to underscores.*
+
 #### Percent encoding characters
 
-Folder names should not use characters that require percent encoding. Those characters may include, but aren't limited to:
+Folder names should not use characters that require percent encoding. Those characters may include, but are not limited to:
 
 - ` ` (spaces)
```

Comment:
Markdown is acting strange with only a space in the code markup. (i.e. no spacing between the code markup and parenthesis, etc.)
![](https://puu.sh/AzfZP/a96fdc7667.png)

Reply:
Hmm, will change `` ` ` `` to `` `Space` `` (this will be noted in the ASC as well)